vault/plugin_catalog_test.go

```diff
@@ -50,8 +50,8 @@ func TestPluginCatalog_CRUD(t *testing.T) {
 	}
 	defer file.Close()
 
-	command := fmt.Sprintf("%s --test", filepath.Base(file.Name()))
-	err = core.pluginCatalog.Set("mysql-database-plugin", command, []byte{'1'})
+	command := fmt.Sprintf("%s", filepath.Base(file.Name()))
+	err = core.pluginCatalog.Set("mysql-database-plugin", command, []string{"--test"}, []byte{'1'})
```

Comment:
Please also an a check to verify correct behavior if both args and command-with-args are given (after fixing above)

Reply:
I've added the test to logical_system_test.go, as the logic to ensure only command or args is specified is in logical_system.go - hope thats okay!